scripts/zkai_stwo_transformer_block_plan.py

```
@@ -0,0 +1,231 @@
+    no_go_ids = _ids(_required_list(plan.get("no_go_criteria"), "no_go_criteria"), "no_go_criteria")
+    _require_superset(no_go_ids, REQUIRED_NO_GO_IDS, "no_go_criteria")
+
+    non_claims = _required_list(plan.get("non_claims"), "non_claims")
+    non_claim_text = "\n".join(str(item).lower() for item in non_claims)
+    for fragment in REQUIRED_NON_CLAIM_FRAGMENTS:
+        if fragment.lower() not in non_claim_text:
+            raise PlanValidationError(f"non_claims is missing: {fragment}")
+
+    commands = _required_list(plan.get("validation_commands"), "validation_commands")
+    if "python3 -m unittest scripts.tests.test_zkai_stwo_transformer_block_plan" not in commands:
+        raise PlanValidationError("validation_commands must include the focused unit test")
```

Ruff (0.15.12)

[warning] Avoid specifying long messages outside the exception class

(TRY003)